tensorflow/core/common_runtime/gpu/gpu_init.h

```diff
@@ -18,13 +18,19 @@ limitations under the License.
 
 #include "tensorflow/core/lib/core/status.h"
 
+#if GOOGLE_CUDA
```

Comment:
Could you implement this as a no-inline function rather than a macro?

Reply:
done, though I am not sure I did exactly what you had in mind...please re-review.

This change was also the cause of the CI failures. The `GPU_PLATFORM_NAME` macro (and now the `GpuPlatformName` function) needs to have a default value (which will be "CUDA" to preserve existing functionality)

Reply:
Looks good, except that you should use 'string' instead of 'std::string'.